install.ps1

```diff
@@ -0,0 +1,78 @@
+# Copyright 2022 The NATS Authors
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+$ErrorActionPreference = 'Stop'
+
+$Version = ""
+if ($args.Length -eq 1) {
+    $Version = $args.Get(0)
+}
+
+$NscInstall = $env:NSC_INSTALL
+$BinDir = if ($NscInstall) {
+    "$NscInstall\bin"
+} else {
+    "$Home\.nats\bin"
+}
+
+$NscZip = "$BinDir\nsc.zip"
+$NscExe = "$BinDir\nsc.exe"
+$Target = 'windows-amd64'
+
+if (Test-Path "$BinDir\nsc.exe") {
+	Write-Host "NSC binary ($NscExe) already exists, exiting script.`r`n"
+	Exit -1
+}
+
+if ( !(Test-Path $BinDir) ) {
+    New-Item $BinDir -ItemType Directory | Out-Null
+	Write-Host "NSC directory ($BinDir) created...`r`n"
+}
+
+# Download the archive with the exe from github
+## GitHub requires TLS 1.2
+[Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
+
+## Uri is different when a version is requested
+if (!$Version) {

```

Comment:
I know that for nsc we don't bake the version in on the zip - but for other things we do - we should make this generic, and attempt both of the file names. To determine the tag on latest take a look at 

https://blog.markvincze.com/download-artifacts-from-a-latest-github-release-in-sh-and-powershell/

Reply:
I'm pulling the information from Phil's [json file](https://get-nats.io/synadia-nats-platforms.json) that has the details we need.